mods/tnt/init.lua

```diff
@@ -384,7 +349,72 @@ minetest.register_craft({
 	}
 })
 
-if minetest.setting_get("log_mods") then
-	minetest.debug("[TNT] Loaded!")
+function tnt.register_tnt(def)
+	local name = ""
+	if not def.name:find(':') then
+		name = "tnt:"..def.name
+	else
+		name = def.name
+		def.name = def.name:sub(def.name:find(':')+1)
+	end
+	if not def.tiles then def.tiles = {} end
+	local tnt_top = def.tiles.top or def.name.."_top.png"
+	local tnt_bottom = def.tiles.bottom or def.name.."_bottom.png"
+	local tnt_side = def.tiles.side or def.name.."_side.png"
+	local tnt_burning = def.tiles.burning or def.name.."_top_burning_animated.png"
```

Comment:
@rubenwardy It's 79 characters long and 83 columns I think I It might be best to leave it as it is after all it fits in the 90 character hard limit.